core/components/atoms/avatarGroup/__stories__/variants/Size.story.jsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { AvatarGroup, Text } from '@/index';
+import { list } from '../AvatarList';
+
+export const size = () => {
+  return (
+    <div className="d-flex">
+      <div className="d-flex flex-column">
+        <Text weight="strong">Regular</Text>
+        <div className="d-flex">
+          <div className="mr-9 mt-4">
+            <AvatarGroup size="regular" list={list.slice(0, 4)} />
+          </div>
+          <div className="mr-9 bg-secondary-lightest p-4" style={{ color: 'var(--secondary-lightest)' }}>
+            <AvatarGroup size="regular" borderColor="" list={list.slice(0, 4)} />
```

Comment:
Instead of using `style` tag please use the color related classes `color-secondary-lightest`

Reply:
Done